docs/content/documentation/_index.md

```diff
@@ -9,14 +9,14 @@ the programming language Java, which defines how a client may access a database.
 It is part of the Java Standard Edition platform and provides methods to query and
 update data in a database, and is oriented towards relational databases.
 
-PostgreSQL JDBC Driver (pgJDBC for short) allows Java programs to connect to a PostgreSQL
+PostgreSQL™ JDBC Driver (pgJDBC for short) allows Java programs to connect to a PostgreSQL™
```

Comment:
IANAL, but I think that https://www.postgresql.org/about/policies/trademarks/ uses **PostgreSQL®** instead of **PostgreSQL™** since it's a _registered trademark_ and a TM (™) sign denotes an unregistered trademark